src/includes/schemaorg/class-wordlift-schemaorg-class-service.php

```diff
@@ -101,6 +101,10 @@ public function schemaorg_class() {
 
 		$json = array_map(
 			function ( $term ) use ( $name_meta_key, $parent_of_meta_key ) {
+				// Ensure $term is valid.
+				if ( ! $term || ! is_object( $term ) || ! isset( $term->term_id ) ) {
+					return null;
+				}
```

Comment:
According to chatgpt this should be enough:

```
if ( ! is_object( $term ) || empty( $term->term_id ) ) {
    return null;
}
```
https://chatgpt.com/share/673e6a02-b8f4-8012-b8f3-a633b4ca1fc6